packages/api/src/shared/log.ts

```diff
@@ -13,3 +14,9 @@ export function debug(msg: string, ...optionalParams: any[]): void {
     console.log(msg);
   }
 }
+
+export function errorToString(err: unknown): string {
+  return err instanceof ZodError
+    ? Object.values(err.flatten().fieldErrors).join("; ")
+    : (err as any)["message"] ?? String(err); // eslint-disable-line @typescript-eslint/no-explicit-any
+}
```

Comment:
`String(error)` when `error` is an instance of `ZodError` results in multi-line logs. This is to prevent that.